pkg/plugin/plugin.go

```diff
@@ -45,7 +46,7 @@ func newRemotePlugin(details *registry.ProviderDetails, alias string, env []stri
 	client := plugin.NewClient(&plugin.ClientConfig{
 		HandshakeConfig: serve.Handshake,
 		VersionedPlugins: map[int]plugin.PluginSet{
-			2: pluginMap,
+			3: pluginMap,
```

Comment:
```suggestion
			3: pluginMap,
			2: pluginMap,
```

You need to support also previous protocol, I think the above should do the trick

Reply:
This wont work here as we will need to get the json back and we removed this code. This is why we release first the providers and then cloudquery core. so users will be able to upgrade to latest provider